client/src/components/BusinessInfo/BusinessInfo.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import $ from 'jquery';
+
+class BusinessInfo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      res_id: 1,
+      status: 'close',
+    };
+  }
+
+  componentDidMount() {
+    this.openStatus();
+  }
+
+  openStatus() {
+    const { restaurant } = this.props;
+    if (restaurant) {
+      const today = new Date();
+      const rightnow = today.getHours();
+      const amOpentime = restaurant.hours.morning.opentime;
+      const amClosetime = restaurant.hours.morning.closetime;
+      const pmOpentime = restaurant.hours.afternoon.opentime;
+      const pmClosetime = restaurant.hours.afternoon.closetime;
+      const inRange = (rightnow <= amClosetime && rightnow >= amOpentime) || (rightnow <= pmClosetime && rightnow >= pmOpentime);
+      if (inRange) {
+        this.setState({
+          status: 'open',
+        });
+      } else {
+        this.setState({
+          status: 'close',
+        });
+      }
+    }
+  }
+
+  renderStatus() {
+    if (this.state.status === "close") {
+      $('span.color-clock').addClass('close-status');
+      return <span className="close-status">Closed now</span>;
+    } else if (this.state.status === "open") {
+      $('span.color-clock').addClass('open-status');
```

Comment:
Don't use jquery to add class. Use a function in your React component that renders the select component with or without that class